docs/root/intro/arch_overview/listener_filters.rst

```diff
@@ -0,0 +1,19 @@
+.. _arch_overview_listener_filters:
+
+Listener filters
+================
+
+As discussed in the :ref:`listener <arch_overview_listeners>` section, listener filters may be
+used to manipulate connection metadata. In fact, Envoy implements :ref:`*use_original_dst* and
```

Comment:
nit: I would probably just delete the "in fact" sentence, I don't think the v1 details are super important and will become much less over time.

Reply:
Deleted.